model/chat.go

```diff
@@ -7,10 +7,11 @@ import (
 
 type Chat struct {
 	gorm.Model
-	UserID   string
-	UserName string
-	Message  string
-	StreamID uint
-	Admin    bool
-	SendTime time.Time
+
+	UserID   string    `gorm:"not null"`
```

Comment:
Did you check if gorm applies these changes to an existing database? What would happen with `null` values?
If not we might need a migration. I did something similar here: https://github.com/TUM-Dev/Campus-Backend/tree/main/server/backend/migration If you want to check it out. 
It basically creates another table that lists which migrations have been applied to the database and runs them in case they are new. Reference: https://github.com/go-gormigrate/gormigrate

Reply:
Yes, the updates applied automatically as there is an automigration happening when starting up the server. Unfortunately, it cannot resolve some conflicts, e.g. when a column that was annotated with non-null still contains null values this causes error. If we want to delete incorrect null entries, we will have to do it manually.